components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -317,6 +319,9 @@ be lost if SCREAM_HACK_XML is not enabled.
       <cosp_frequency_units valid_values="steps,hours">hours</cosp_frequency_units>
     </cosp>
 
+    <!-- Turbulent Mountain Stress -->
+    <tms inherit="atm_proc_base"/>
```

Comment:
If this was not in master, how were noel/chris able to add tms via atmchange?

Reply:
That is a good question. It doesn't look like on the cess-production branch it is included either.

Reply:
Looking back through the eamxx-simulation chat, I think that Noel encountered the error of no TMS registered and commented out `./atmchange physics::mac_aero_mic::atm_procs_list="tms,..."` so most likely running without TMS, and Oksana was most likely running with a version of the code before TMS was removing .